packages/mobile/src/screens/edit-track-screen/screens/AccessAndSaleScreen.tsx

```diff
@@ -116,6 +117,8 @@ export const AccessAndSaleScreen = () => {
 
   const isInitiallyPublic =
```

Comment:
since we have a lot of similar logic around this piece, and the one on [line 132](https://github.com/AudiusProject/audius-protocol/pull/6409/files#diff-005ccff4562b181dddf788572d79b8e952ea95c9866d026204d6a417bd8b72e1R132), both on web and mobile, I'm going to refactor them.

thinking of using a hook e.g. `useAccessAndRemixSettings` that will return whether each of the Access & Sale + Remix options should be enabled or not.

other thoughts are welcome.

Reply:
sounds good to me!